packages/javascript-sdk/src/token-manager/index.ts

```diff
@@ -118,8 +118,9 @@ abstract class TokenManager {
      */
     const verifier = PKCE.createVerifier();
     const state = PKCE.createState();
+    const { forceRenew, login, query, ...config } = options;
```

Comment:
unsure if theres a better way to remove the lint warnings on unused variables but this felt like the cleanest option.

Reply:
Ok, think I understand this now after reading through for a little bit. Not sure how we can remove the lint warnings but will have a think. What you're doing makes sense though.

Reply:
I think we need to pass `query` into the `authorizeUrlOrptions` as well, yeah? It's used here: https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/packages/javascript-sdk/src/oauth2-client/index.ts#L53.

Reply:
Yes I think you're right, good spot. Yeah it looks like `GetTokensOptions` defines query on top of `ConfigOptions`.